tests/algorithms/pytorch/test_fed_avg.py

```diff
@@ -46,7 +46,7 @@ def __init__(
                 model=perceptron,
                 index_generator=nig,
                 dataset=numpy_torch_dataset,
-                use_gpu=False,
+                disable_gpu=True,
```

Comment:
I don't think this is required 🤔 
and same for the tests of other implementations